public/app/percona/dbaas/components/DBCluster/EditDBClusterPage/EditDBClusterPage.utils.ts

```diff
@@ -47,19 +52,38 @@ export const generateUID = (): string => {
   return firstPart + secondPart;
 };
 
-export const getEditInitialValues = (selectedDBCluster: DBCluster): EditDBClusterFormValues => {
+export const getDBClusterConfiguration = async (selectedCluster: DBCluster): Promise<DBClusterPayload | undefined> => {
+  try {
+    const dbClusterService = newDBClusterService(selectedCluster.databaseType);
+    const result = await dbClusterService.getClusterConfiguration(selectedCluster);
+    return result;
+  } catch (e) {
+    logger.error(e);
+  }
+  return;
```

Comment:
The return is probably redundant here.

Reply:
![image](https://user-images.githubusercontent.com/90199600/211520084-d5a35010-922b-4008-9a95-b5475673b221.png)